src/components/WebexMember/WebexMember.jsx

```diff
@@ -22,16 +22,21 @@ export default function WebexMember({
   displayStatus,
 }) {
   const {firstName, lastName} = usePerson(personID);
+  const me = useMe();
   const members = useMembers(destinationID, destinationType);
   const member = members
     .find((itemMember) => itemMember.personID === personID);
 
   const isMuted = member && member.muted;
+  const showMe = me.ID === personID && destinationType === DestinationType.MEETING;
 
   return (
     <div className={`${WEBEX_COMPONENTS_CLASS_PREFIX}-member`}>
       <WebexAvatar personID={personID} displayStatus={displayStatus} />
-      <div className="member-name">{`${firstName} ${lastName}`}</div>
+      <div className="description">
```

Comment:
Is this `description` div necessary?
Can the subtext just be part of the `member-name` div?

Reply:
We agree with the team to use this structure, we consider that member-name should contain the only the name, after that will come 'representor', 'you', 'organization name'

Reply:
Ok, we can keep the structure if the team thinks that's best. I'm not a fan of the class name, not sure it makes sense to me why it's called `description`? Maybe a better name would be `details`